test/unittests/state_transition_eof_create_test.cpp

```diff
@@ -2089,7 +1989,7 @@ TEST_F(state_transition, txcreate_failure_after_txcreate_success)
         sstore(0, txcreate().initcode(keccak256(init_container)).salt(Salt)) +
         sstore(1, txcreate().initcode(keccak256(init_container)).salt(Salt)) +  // address collision
         sstore(2, returndatasize()) + sstore(3, 1) + OP_STOP;
-    const auto factory_container = eof_bytecode(factory_code, 5).container(init_container);
```

Comment:
What is this change about?

Reply:
It was a bug in original test, so it was failing for the wrong reason I guess.